src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("numbers 크기 확인")
+    public void numbers_크기_확인() throws Exception{
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("numbers 의 요소 확인")
+    public void numbers_요소_확인(int expected) throws Exception{
```

Comment:
```suggestion
    public void numbers_요소_확인(int expected) throws Exception {
```
코딩 컨벤션에 맞춰 작성해주시면 좋을 것 같습니다 :) 👀
만약 IntelliJ를 사용하신다면 Cmd + Option + L로 손쉽게 사용하실 수 있습니다 :)